less/index.less

```diff
@@ -49,11 +49,214 @@ table {
 }
 
 /* Set every element's box-sizing to border-box */
-* {
-    box-sizing: border-box;
+
+// variables 
+
+@font-stack: 'Roboto', sans-serif;
```

Comment:
great use of font-stack!

Reply:
Thank you! Yea it really helped and was a good learning from Josh